scripts/image_versions.src

```diff
@@ -17,3 +17,4 @@
 # See https://github.com/department-of-veterans-affairs/abd-vro/wiki/Container-Image-Versions
 # Wed Jul 12 17:45:45 UTC 2023 -- v3.3.1
 # Mon Jul 31 17:42:17 UTC 2023 -- v3.4.0
+# Tue Aug 15 09:22:32 PDT 2023 -- v3.4.0
```

Comment:
```suggestion
```

Reply:
What is the change for @yoomlam ?

Reply:
There should be no changes to this file for your PR.